data/html-parser/remove-paragraphs-from-definition-lists.ts

```diff
@@ -1,19 +1,20 @@
-export const removeParagraphsFromDefinitionListsAndMerge = (cheerioNodes: cheerio.Selector) =>
-  cheerioNodes('dl').each((_i: number, elem: cheerio.Element) => {
+export const removeParagraphsFromDefinitionListsAndMerge = (cheerioNodes: cheerio.Selector) => {
+  return cheerioNodes('dl').each((_i: number, elem: cheerio.Element) => {
```

Comment:
This return isn't strictly needed, as it's mutating elements (not sure that each will return anything so it doesn't change signature of function I think).